typescript/sdk/src/hook/EvmHookReader.ts

```diff
@@ -125,13 +121,13 @@ export class EvmHookReader implements HookReader {
           );
       }
     } catch (e) {
-      this.logger.debug(
-        `Failed to derive ${onchainHookType} hook (${address}): ${e}`,
+      this.logger.error(
+        `Failed to derive ${onchainHookType} hook (${address}):`,
+        e,
       );
-    } finally {
```

Comment:
If you remove the `finally`, doesnt this mean that the provider logging will always be `silent` downstream? in other words, if there is no error, then provider logs are always silent.

Reply:
ah yep thanks for catching this– moved this out and it should always be executed now